### PR TITLE
CR015: Harmonizing enums with TPEG2 PTS tables

### PR DESCRIPTION
**Pull request by hennoit**
Most TPEG Pti enumerations replaced/extended with TPEG2 Pts values. Numerical values removed from enums; only textual values retained. No changes yet to the various _xxxMode_ enumerations. 
See [CR015 on basecamp](https://3.basecamp.com/3256016/buckets/9672657/uploads/1952413525) for the documentation.

### CODE REVIEW

File: xsd/siri_model/siri_time-v2.0.xsd [hunk 1]
```diff
@@ -34,19 +43,18 @@
      <Syntax>http://www.w3.org/2001/XMLSchema</Syntax>
      <Description>XML schema, W3C Recommendation 2001</Description>
     </Format>
-    <Identifier>{http://www.siri.org.uk/schema/2.0/xsd/siri_model/}siri_time-v1.2.xsd</Identifier>
+    <Identifier>{http://www.siri.org.uk/schema/2.0/xsd/siri_model/}siri_time-v2.0.xsd</Identifier>
     <Language>[ISO 639-2/B] ENG</Language>
     <Publisher>Kizoom, 109-123 Clifton Street, London EC4A 4LD </Publisher>
```
Comment (tim-rivett) [inline xsd/siri_model/siri_time-v2.0.xsd]: <Publisher>CEN</Publisher>

File: xsd/siri_model/siri_time-v2.0.xsd [hunk 2]
```diff
@@ -189,63 +197,129 @@ Rail transport, Roads and road transport
...
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="wednesday">
+    <xsd:annotation>
+     <xsd:documentation>TPEG Pti34_3, Wednesday</xsd:documentation>
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="thursday">
+    <xsd:annotation>
+     <xsd:documentation>TPEG Pti34_4, Thusrday</xsd:documentation>
```
Comment (tim-rivett) [inline xsd/siri_model/siri_time-v2.0.xsd]: <xsd:documentation>TPEG Pti34_4, Thursday</xsd:documentation>

File: xsd/siri_model/siri_situationAffects-v2.0.xsd [hunk 3]
```diff
@@ -64,20 +77,19 @@
     <Language>[ISO 639-2/B] ENG</Language>
     <Publisher>Kizoom, 109-123 Clifton Street, London EC4A 4LD</Publisher>
```
Comment (tim-rivett) [inline xsd/siri_model/siri_situationAffects-v2.0.xsd]: <Publisher>CEN</Publisher>

File: xsd/siri_model/siri_time-v2.0.xsd (hunk 1 above)
Comment (hennoit) [reply xsd/siri_model/siri_time-v2.0.xsd]: Changed in next commit

File: xsd/siri_model/siri_time-v2.0.xsd (hunk 2 above)
Comment (hennoit) [reply xsd/siri_model/siri_time-v2.0.xsd]: Corrected in next commit

File: xsd/siri_model/siri_situationAffects-v2.0.xsd (hunk 3 above)
Comment (hennoit) [reply xsd/siri_model/siri_situationAffects-v2.0.xsd]: Changed in next commit

**Pull request merged**